docs/service-mesh/05-05-production-profile.md

```diff
@@ -22,18 +22,108 @@ The production profile introduces the following changes to the Istio Service Mes
 As the production profile is configured with increased performance it mind, the recommend system setup is different:
 
 | Requirement | Development setup | Production setup|
-|:--- | :--- | :--- | 
-| vCPU | 4 | 8 | 
-| RAM | 16 | 16/32 | 
-| Example machine type (GKE) | `n1-standard-4` | `n1-standard-8` or `c2-standard-8` | 
-| Example machine type (AKS) | `Standard_D4_v3` | `Standard_F8s_v2` or `Standard_D8_v3` | 
-| Size | 3 | 3-5 | 
+|:--- | :--- | :--- |
+| vCPU | 4 | 8 |
+| RAM | 16 | 16/32 |
+| Example machine type (GKE) | `n1-standard-4` | `n1-standard-8` or `c2-standard-8` |
+| Example machine type (AKS) | `Standard_D4_v3` | `Standard_F8s_v2` or `Standard_D8_v3` |
+| Size | 3 | 3-5 |
 
 ## Use the production profile
 
+>**CAUTION:** Istio is changing the installation options. Helm-based configuration is deprecated in favor of the new IstioControlPlane API. Please keep in mind that Helm overrides will be no longer supported in future Istio releases. Refer to [IstioControlPlane](https://istio.io/docs/reference/config/istio.operator.v1alpha12.pb) page for more information.
```

Comment:
```suggestion
>**CAUTION:** Istio is changing the installation options. Helm-based configuration is deprecated in favor of the new IstioControlPlane API. Please keep in mind that Helm overrides will no longer be supported in future Istio releases. Refer to [IstioControlPlane](https://istio.io/docs/reference/config/istio.operator.v1alpha12.pb) page for more information.
```
Also, I wonder about this sentence:
> Istio is changing the installation options. 

It creates a risk of the document getting outdated once Istio's done with changing the installation options if you fail to remember to update it. 
Maybe rephrase it to something like `Due to changes in the installation options in Istio, Helm-based configuration is now deprecated in favor of the new IstioControlPlane API.`?

Reply:
Suggestion applied, thanks.